metashare/api/sf_org_changes.py

```diff
@@ -0,0 +1,59 @@
+from cumulusci.salesforce_api.utils import get_simple_salesforce_connection
+
+from .github_context import extract_owner_and_repo, local_github_checkout
+from .sf_run_flow import BaseCumulusCI, refresh_access_token
+
+
+def get_salesforce_connection(
```

Comment:
I would probably turn this into a method of the ScratchOrg model. All of the inputs can be derived from the ScratchOrg or its project.

org_name should be stored on the ScratchOrg when it's created and used from there, so that this is usable for qa orgs too.

The base_url update should be an optional parameter (so default is the standard REST API but you can pass the param if you want the tooling API)

Reply:
@davisagli where is `org_name` at creation time? I'm guessing somewhere in the `scratch_org_config`, but I am not sure. I've been going through the various bits of documentation [here](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs.htm) to no avail yet.

Reply:
Never mind, for our purposes it's always dev.

Reply:
So I'm trying to encapsulate the logic that interacts with SF in its own modules, to make it easier to change when/if changes arrive in cumulusci. I'm happy to put a method on ScratchOrg that calls out to this, but I'd like to keep the meat of the logic in this module.